src/test/java/com/exasol/mavenprojectversiongetter/MavenProjectVersionGetterTest.java

```diff
@@ -0,0 +1,13 @@
+package com.exasol.mavenprojectversiongetter;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.jupiter.api.Test;
+
+class MavenProjectVersionGetterTest {
+    @Test
+    void test() {
+        assertThat(MavenProjectVersionGetter.getCurrentProjectVersion(), equalTo("0.1.0"));
+    }
+}
```

Comment:
This is going to fail with version update of the project. 

You can consider adding separate pom for testing (similar to the ones in our maven plugins).

Reply:
Yea I thought about it too.
The issue is, I need to 
* create the project
* move the test code there
* compile it
* run it
* validate the result
I'm not sure if it's worth it since this test is so easy and readable in comparison.